packages/nx/src/plugins/js/lock-file/pnpm-parser.spec.ts

```diff
@@ -183,12 +183,12 @@ describe('pnpm LockFile utility', () => {
         // this is our pruned lock file structure
         const prunedGraph = pruneProjectGraph(graph, appPackageJson);
         expect(Object.keys(prunedGraph.externalNodes).length).toEqual(
-          864 + 117 // peer cypress adds additional 117 deps
+          864 + 119 // peer cypress adds additional 119 deps
         );
 
         // this should not fail
         expect(() =>
-          stringifyPnpmLockfile(prunedGraph, lockFile, appPackageJson)
```

Comment:
Currently on `main`, if actually try to `pnpm install --frozen-lockfile` using the stringified lockfile, you get a ` ERR_PNPM_LOCKFILE_MISSING_DEPENDENCY `.